ci/publish.sh

```diff
@@ -1,7 +1,12 @@
 #!/usr/bin/env bash
 
 # This is an entrypoint for CI build step, don't change it's relative path(name)
+# force Envs be set up
+set -u
 
 source $(dirname $0)/_main.sh
-
-runInBuilder "./gradlew -p subprojects publishRelease ${GRADLE_ARGS} --no-parallel --stacktrace"
+readonly STAGING_PROFILE_ID="-Pavito.ossrh.stagingProfileId=${$OSSRH_STAGING_ID}"
+readonly USER="-Pavito.ossrh.user=$OSSRH_USER"
+readonly PASSWORD="-Pavito.ossrh.password=$OSSRH_PASSWORD"
+readonly CREDENTIALS = "${STAGING_PROFILE_ID} ${USER} ${PASSWORD}"
```

Comment:
Move from TeamCity configuration to code. Because TeamCity conf was broken after moving `gradle.properties` file

Reply:
you moved only 3 out of 6 properties from teamcity; why so?